auditbeat/auditbeat.reference.yml

```diff
@@ -86,12 +86,22 @@ auditbeat.modules:
 # sub-dictionary. Default is false.
 #fields_under_root: false
 
-# Internal queue size for single events in processing pipeline
-#queue_size: 1000
-
-# The internal queue size for bulk events in the processing pipeline.
-# Do not modify this value.
-#bulk_queue_size: 0
+# Internal queue configuration for buffering events to be published.
+#queue:
+  # Queue type by name (default 'mem')
+  #mem:
+    # Max number of events to store in the queue.
+    #events: 4096
```

Comment:
What happens when this is reached? Flush happens? Thinking if there should be `flush.min`, `flush.max` and `flush.timeout` or similar?